interface/js/pokedex.js

```diff
@@ -0,0 +1,153 @@
+          dropdown.classList.add("dropdown-menu");
+          dropdown.style.padding = "0";
+
+          // Create a table for the dropdown
+          const table = document.createElement("table");
+          const headerRow = document.createElement("tr");
+          const methodHeader = document.createElement("th");
+          methodHeader.textContent = "Method";
+          headerRow.appendChild(methodHeader);
+          const levelsHeader = document.createElement("th");
+          levelsHeader.textContent = "Levels";
+          headerRow.appendChild(levelsHeader);
+          const rateHeader = document.createElement("th");
+          rateHeader.textContent = "Rate";
+          headerRow.appendChild(rateHeader);
+          table.appendChild(headerRow);
+          table.style.marginLeft = "1em";
+          table.style.marginRight = "1em";
+          table.style.tableLayout = "auto";
+
+          // Set values for each encounter
+          encounters.forEach((encounter) => {
+            const valuesRow = document.createElement("tr");
+            const methodCell = document.createElement("td");
+            methodCell.style.whiteSpace = "nowrap";
+            methodCell.textContent = getMethod(encounter.encounter_type);
+            const levelsCell = document.createElement("td");
+            levelsCell.style.whiteSpace = "nowrap";
+            levelsCell.textContent = encounter.levels;
+            const rateCell = document.createElement("td");
+            rateCell.style.whiteSpace = "nowrap";
+            rateCell.textContent = encounter.rate;
+            valuesRow.appendChild(methodCell);
+            valuesRow.appendChild(levelsCell);
+            valuesRow.appendChild(rateCell);
+            table.appendChild(valuesRow);
+          });
+
+          dropdown.appendChild(table);
+          div.appendChild(pill);
+          div.appendChild(dropdown);
+          locationCell.appendChild(div);
+        });
+      } else {
+        locationCell.textContent = "";
+      }
+
+      // append the cells to the row
+      row.appendChild(dexIDCell);
+      row.appendChild(imgCell);
+      row.appendChild(nameCell);
+      row.appendChild(locationCell);
+
+      //add the row to the table #pokedex
+      tableBody.appendChild(row);
+    });
+  });
+}
+
+dexEntries();
+
+//mess tbh, but it formats the encounter type nicer
+function getMethod(method) {
+  const methodMap = {
+    walking: "Walking",
+    walk: "Walking",
+    fishing_old: "Old Rod",
+    fishing_good: "Good Rod",
+    fishing_super: "Super Rod",
+    special: "Special Encounter",
+    deepsand: "Deep Sand",
+    rocksmash: "Rock Smash",
+    surfing: "Surfing",
+    surf: "Surfing",
+    grass: "Grass",
+    swarm: "Swarm",
+    trade: "Trade",
+    gift: "Gift",
+    roam: "Roaming",
+    underwater: "Dive Underwater",
+    wailmerpail: "Wailmer Pail",
+    hidden: "Hidden",
+    starter: "Starter",
+  };
+
+  return methodMap[method] || "";
+}
```

Comment:
And something like this here:
```
window.setInterval(function() {
    encounter_log();
}, 1000);

```

Reply:
Also probably remove import for dashboard js, alternatively you could probably just add an if check in dashboard.js for the encounter log, which would probably be cleaner. I think it fails when not finding it in the Pokedex, then causing the rest not to run.